include/deal.II/distributed/shared_tria.h

```diff
@@ -224,6 +224,27 @@ namespace parallel
         construct_multigrid_hierarchy = 0x8,
       };
 
+      /**
+       * Flags that determine which type of subdomain ID will be returned when
+       * calling CellAccessor::subdomain_id() on active cells.
+       */
+      enum SubdomainMask
```

Comment:
below you always write `SubdomainMask::default_ids`, but you don't actually need the `SubdomainMask::` part unless you want to (which I would approve of) change this to
```suggestion
      enum class SubdomainMask
```